### ISSUE TITLE
10586 support

### ISSUE THREAD

**Issue by Syn-McJ** [XamlBrewer/UWP-Composition-BackDrop-Control#1]
Hi. Very useful control you have here and I'd love to use it in my project. Problem is that we still have some users on 10586 and this control will cause exceptions if I just leave it in layout. I could work around this, of course, but I think it would be better to solve this problem once on lower level so that later users don't have to.

I have made some modifications which you can check here: https://github.com/Syn-McJ/UWP-Composition-BackDrop-Control/blob/threshold2-support/XamlBrewer.Uwp.BackDrop/BackDrop.cs

Basically it's just matter of runtime checks for ApiInformation.IsApiContractPresent. Do you think we can merge these changes in your library?
